member.php

```diff
@@ -2028,7 +2028,7 @@
 				$captcha = $login_captcha->build_hidden_captcha();
 			}
 		}
-		elseif(in_array($login_captcha->type, array(2, 4, 5)))
+		elseif(in_array($login_captcha->type, array(2, 4, 5, 8)))
```

Comment:
Type 2 is long gone. I guess its an unintentional left out. Please change it to `array(4, 5, 8)`